src/script/components/TitleBar/index.tsx

```diff
@@ -0,0 +1,332 @@
+}) => {
+  const {
+    is1to1,
+    isRequest,
+    isActiveParticipant,
+    isGroup,
+    hasExternal,
+    hasDirectGuest,
+    hasService,
+    hasFederatedUsers,
+    firstUserEntity,
+    hasLegalHold,
+    display_name: displayName,
+    verification_state: verificationState,
+  } = useKoSubscribableChildren(conversation, [
+    'is1to1',
+    'isRequest',
+    'isActiveParticipant',
+    'isGroup',
+    'hasExternal',
+    'hasDirectGuest',
+    'hasService',
+    'hasFederatedUsers',
+    'firstUserEntity',
+    'hasLegalHold',
+    'display_name',
+    'verification_state',
+  ]);
+
+  const {isVisible: isPanelVisible} = useKoSubscribableChildren(panelViewModel, ['isVisible']);
+
+  const {isActivatedAccount} = useKoSubscribableChildren(userState, ['isActivatedAccount']);
+  const {joinedCall} = useKoSubscribableChildren(callState, ['joinedCall']);
+  const {isVideoCallingEnabled} = useKoSubscribableChildren(teamState, ['isVideoCallingEnabled']);
+
+  const badgeLabelCopy = useMemo(() => {
+    if (is1to1 && isRequest) {
+      return '';
+    }
+
+    const translationKey = generateWarningBadgeKey({
+      hasExternal,
+      hasFederated: hasFederatedUsers,
+      hasGuest: hasDirectGuest,
+      hasService,
+    });
+
+    if (translationKey) {
+      return t(translationKey);
+    }
+
+    return '';
+  }, [hasDirectGuest, hasExternal, hasFederatedUsers, hasService, is1to1, isRequest]);
+
+  const hasCall = useMemo(() => {
+    const hasEntities = !!joinedCall;
+    return hasEntities && matchQualifiedIds(conversation.qualifiedId, joinedCall.conversationId);
+  }, [conversation, joinedCall]);
+
+  const showCallControls = ConversationFilter.showCallControls(conversation, hasCall);
+
+  const supportsVideoCall = conversation.supportsVideoCall(callingRepository.supportsConferenceCalling);
+
+  const conversationSubtitle = is1to1 && firstUserEntity.isFederated ? firstUserEntity.handle ?? '' : '';
+
+  const shortcut = Shortcut.getShortcutTooltip(ShortcutType.PEOPLE);
+  const peopleTooltip = t('tooltipConversationPeople', shortcut);
+
+  const showDetails = useCallback(
+    (addParticipants: boolean): void => {
+      const panelId = addParticipants
+        ? PanelViewModel.STATE.ADD_PARTICIPANTS
+        : PanelViewModel.STATE.CONVERSATION_DETAILS;
+      panelViewModel.togglePanel(panelId, {entity: conversation});
+    },
+    [conversation, panelViewModel],
+  );
+
+  const showAddParticipant = useCallback(() => {
+    if (!isActiveParticipant) {
+      return showDetails(false);
+    }
+
+    if (isGroup) {
+      showDetails(true);
+    } else {
+      amplify.publish(WebAppEvents.CONVERSATION.CREATE_GROUP, 'conversation_details', firstUserEntity);
+    }
+  }, [firstUserEntity, isActiveParticipant, isGroup, showDetails]);
+
+  useEffect(() => {
+    // TODO remove the titlebar for now to ensure that buttons are clickable in macOS wrappers
+    window.setTimeout(() => $('.titlebar').remove(), TIME_IN_MILLIS.SECOND);
```

Comment:
```suggestion
    window.setTimeout(() => document.querySelector('.titlebar').remove(), TIME_IN_MILLIS.SECOND);
```
Let's try to avoid using jQuery for this